.github/workflows/publish-paradedb.yml

```diff
@@ -93,6 +93,10 @@ jobs:
             type=raw,value=${{ steps.version.outputs.tag }},enable=${{ matrix.pg_version == 16 }}
             type=raw,value=${{ steps.version.outputs.version }},enable=${{ matrix.pg_version == 16 }}
             type=raw,value=latest,enable=${{ matrix.pg_version == 16 }}
+            type=raw,value=postgres${{ matrix.pg_version }}
```

Comment:
I'm still pretty against this one... There is no ParadeDB version in it. How can we release a version that has no ParadeDB version.

The other ones seem fine re: my other comments